storage/netapp_e/na_eseries_storage_system.py

```diff
@@ -212,7 +224,7 @@ def main():
                 changed = True
             if array_detail['wwn'] != array_wwn and array_wwn is not None:
                 module.fail_json(
-                    msg='It seems you may have specified a bad WWN. The storage system ID you specified, %s, " + '
+                    msg='It seems you may have specified a bad WWN. The storage system ID you specified, %s, ' +
                         'currently has the WWN of %s' % (ssid, array_detail['wwn']))
```

Comment:
The plus will mess things up.  Like C, python will combine adjacent strings automatically into a single string literal.  Using the plus, otoh, will make python create two separate string literals and then concat them as a third string.  So the plus is unnecessary and makes things slightly less efficient.

In this case, there's a further wrinkle.  We're doing string formatting using %.  because of that, precedence rules come into play.  Python will first combine adjacent literal strings, then do percent formatting, and finally concatenate strings via plus.  That means that with the plus, when we get to the percent formatting operation we don't yet have a string with both %s's in it.  So we'd get a TypeError:

```
TypeError: not all arguments converted during string formatting
```

Removing the plus will resolve that.